Source/quests.cpp

```diff
@@ -652,6 +653,8 @@ void SetReturnLvlPos()
 		ReturnLvl = quests[Q_SCHAMB]._qlevel;
 		ReturnLvlT = DTYPE_CATACOMBS;
 		break;
+	case SL_MAZE:
+		break;

```

Comment:
Unrelated

Reply:
yeah, just got rid of a warning to make it cleare to see if i caused any issues